__test__/option.spec.ts

```diff
@@ -1,20 +1,23 @@
+import ChowChow from "../src";
```

Comment:
Do you mind set the prettier to prefer single quote over double quote since that's the common pattern in the code base already.